### DISCUSSION TITLE
# OPi 5's down with todays update

### DISCUSSION THREAD

**Discussion by fyptog** [Joshua-Riek/ubuntu-rockchip#1009] [General]
Hello, I need some help recovering my 3x Orange Pi 5s that don't boot after an update today. 
I rebooted after updating and got an (initramfs) prompt on all three machines, the nvme ssd is not anywhere to be seen from the prompt or when I reboot Ubuntu with an SD card.
So, thinking that the spi must be corrupted, I used Disks to reflash spi and rebooted with my SD, and nvme was then visible (using Disks) . Then removed the SD and rebooted only to have the same scenario repeat ie. (initramfs) prompt w/ no ssd registering - ad infinitum. It's a bad scene.
I was wanting some help with how to undo the last update and/or remove the last kernel.
Any helpers?
Orange Pi5 16GB, Ubuntu 24.04 desktop, 240GB nvme ssd

**Comment by Joshua-Riek** [comment]
Do you have boot logs?

**Comment by fyptog** [comment]
I don't know how to get them, or if they still exist - I have, just now, switched the kernels in extlinux.conf file so the previous kernel (6.1.0.1023) boots. 
Happy reproduce the problem and pass you the logs if you point me in the right direction.

**Reply by Joshua-Riek** [reply]
Connect a serial console to the board, and you will see the logs.

Are you using a sata ssd?

**Comment by fyptog** [comment]
Using only nvme ssd. I have 2 different board revisions (V1.2 & V1.2.1) and the ssd issue is only present on V1.2. However none boot
I have attached the log I obtained using serial monitor CuteCom, but it makes no sense. Maybe it does to you though.
[cutecom.log](https://github.com/user-attachments/files/16755342/cutecom.log)

**Comment by fyptog** [comment]
another log using moserial
[OPi5-16GB-V1.2.1-hex.log](https://github.com/user-attachments/files/16755450/OPi5-16GB-V1.2.1-hex.log)

**Reply by Joshua-Riek** [reply]
A kernel update with the fix was released a few minutes ago.

**Reply by unevoljitelj** [reply]
opi5 also went to intramfs yestarday, been trying to fix it but cant, mine says uuid does not exist, drive is still there intact with same uuid that it says does not exist (nvme)

hope its fine posting here and not opening new post
not sure if my problem is the same as OP here, but any way to apply this fix without reinstalling? did try switching l0 to l1 to l2 in extlinux.conf but doesnt work

edit: nvm, fixed it, it was a kernel issue, but before i knew that i tried fixing drive/uuid and it changed, now boots again

**Comment by fyptog** [comment]
Legend. Thank you - all is resolved.

**Comment by fyptog** [comment]
I am wondering if those logs I sent are any good. Did I succeed?
